cms/envs/devstack.py

```diff
@@ -99,3 +99,9 @@ def should_show_debug_toolbar(_):
 
 # Dummy secret key for dev
 SECRET_KEY = '85920908f28904ed733fe576320db18cabd7b6cd'
+
+XBLOCK_SETTINGS = {
+    "VideoDescriptor": {
+        "licensing_enabled": True
```

Comment:
I disagree - this is devstack and the point is to explicitly turn it on in devstack. See https://github.com/edx/edx-platform/blob/master/cms/envs/devstack.py#L82

I will move this up to that point in devstack.py though so it's clear.